auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -23,7 +23,7 @@
           large
           color="error"
           data-test="dialog-ok-button"
-          @click="$refs.errorDialog.close()"
+          @click="errorDialogComponent.close()"
```

Comment:
Interesting that this doesn't need to be `errorDialogComponent.value.close()` (as it does in the script below). Oh well, this works.